tools/themis/src/utils.rs

```diff
@@ -12,8 +12,6 @@ macro_rules! _warn {
     }
 }
 
-pub(crate) use _warn as warn;
```

Comment:
the `_warn` is deadcode now? `use _macro as macro` is a work-around against language-level ~~bug~~ noble historical accident where macros don't use normal name resolution rules.

Reply:
I think what he meant was we no longer use the macro at all, it's fine to get rid of it entirely.

Reply:
Ah I misunderstood.  Will remove.